genai/example_test.go

```diff
@@ -1,5 +1,6 @@
-// DO NOT EDIT THIS FILE -- it is automatically generated.
-// See internal/cmd/gen-examples for details.
+// DO NOT EDIT THIS FILE -- it is automatically generated from
```

Comment:
We have a standard form for these: "// This file was generated by <program>. DO NOT EDIT."

Reply:
Done.